docs/src/content/docs/advanced.md

```diff
@@ -530,6 +530,8 @@ While the schema permits you to use composition in any way you like, it’s good
 
 `x-enum-descriptions` and `x-enum-varnames` are each expected to be list of items containing the same number of items as enum. The order of the items in the list matters: their position is used to group them together.
 
+Alternatively you can use `x-enumNames` and `x-enumDescriptions`.
```

Comment:
💯 Great docs add!

Reply:
```suggestion
Alternatively you can use `x-enumNames` and `x-enumDescriptions` ([docs](LINK TO DOCS HERE)).
```

It’d be nice to link to wherever this is outlined in the NSwag docs. Do you have a good reference for this? I couldn’t find one easily and you’d know better than I.